src/platform/plugins/shared/content_management/common/rpc/types.ts

```diff
@@ -16,10 +16,12 @@ export interface ProcedureSchemas {
 
 export type ItemResult<T = unknown, M = void> = M extends void
   ? {
-      item: T;
+      id: string;
```

Comment:
This change might break types for other applications such as Lens and Maps. Hopefully all these apps will soon switch to this `ItemResult`, but we may have to create a new `DashboardItemResult` type in the meantime.